packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_list.dart

```diff
@@ -106,52 +107,195 @@ class _ListControlPane extends StatelessWidget {
 }
 
 class _SnapshotListTitle extends StatelessWidget {
-  const _SnapshotListTitle({Key? key, required this.item}) : super(key: key);
+  const _SnapshotListTitle({
+    Key? key,
+    required this.item,
+    required this.index,
+    required this.selected,
+    required this.editIndex,
+    required this.onEdit,
+    required this.onEditingComplete,
+    required this.onDelete,
+  }) : super(key: key);
 
   final SnapshotItem item;
 
+  final int index;
+
+  final bool selected;
+
+  final ValueListenable<int?> editIndex;
```

Comment:
Can you add doc comment for the new members. Some are not obvious (editIndex, onEdit vs onEditComplete)

Reply:
done